docs/platform/what-is-tizen/versions/tizen-5-5-m2.md

```diff
@@ -0,0 +1,64 @@
+![img](media/5.5_3_Lottie.png)
+
+- **Multi-assistant Framework** :
+It aims to support various AI assistants at the same time for better usability. The framework defines a common interface for making an assistant of the multi-assistant framework. It also provides an expandable wake-up engine that recognizes invocation keyword for each assistant and supports to add a new keyword.
+
+- **Multi-device distributed web engine** :
+It is designed for both memory resource requirements mitigation and JavaScript execution speed enhancement to overcome the resource constraints of the low-end devices. With the prior multi-process design of the web engine, the memory usage of the renderers becomes significant as multiple tabs are created. CPU resource consumption of the renderer processes is also substantial as web applications get complicated.
+To manage hardware resources and to make resource usage predictable in low-end devices, the multi-device distributed web engine introduces features for offloading renderer processes over the network to resource-free devices while the browser process runs on the local low-end device.
+
+- **User Awareness Framework** :
+Smart devices could provide useful services based on the user presence context. Tizen 5.5 supports User Awareness Framework based on multiple sensors. Currently, Wi-Fi, BLE, motion, and light sensors are supported and it could be extended to support other types of sensors based plug-in architecture.
+
+## On Device AI
+
+- **Machine Learning (ML)** : A new Tizen domain, Machine Learning (ML) is introduced along with its ML Inference API sets: “Pipeline ML API” and “Single ML API”. ML Inference API sets allows application developers to apply their machine learning mechanisms or neural network models easily and efficiently. Especially if the intelligence application has multiple sensors, multiple neural network models, and complex topology of data paths, Pipeline ML API can help implement the system efficiently with less time and effort. If developers need to use neural network models preloaded with the platform, APIs in Media Vision are appropriate because ML Inference APIs require developers to supply neural network models. In other words, for predefined special functions, Media Vision is appropriate and for general or custom functions, ML Inference is appropriate.
+ML API sets are based on [NNStreamer](https://github.com/nnsuite/nnstreamer) that allows construction of the GStreamer pipelines that consists of the conventional GStreamer media plugins and the neural network plugins. There are plans to expand its functionality to on-device training in the later releases of Tizen. The ML API set have two API subsets, Pipeline ML API and Single ML API. ML API sets of the current version of Tizen support machine learning inferences only. In ML API sets, there are two API subsets: Pipeline ML API set and Single ML API set.
```

Comment:
@jhchoilee 
Two things and I shall approve the PR: 
1. Please verify the the image alignment: line 18,19, Ensure that the images are inline with the text. 

![align](https://user-images.githubusercontent.com/43839737/70600938-02802700-1c17-11ea-8e7d-8abfc9686bf8.PNG)

If they are, kindly ignore then. There may be issues with GIT rendering. 

2. Sorry, but kindly add a line space for each paragraph break to render. 
Before line 43
Before line 44
Before line 45
Before line 46
Before line 47 

![align1](https://user-images.githubusercontent.com/43839737/70600980-188de780-1c17-11ea-8b40-03f44ed37098.PNG)

Reply:
I commit new one "1)image align 2)add a line space for each paragraph break"